robottelo/cli/base.py

```diff
@@ -132,26 +132,26 @@ def delete(cls, options=None, timeout=None):
         return cls.execute(cls._construct_command(options), ignore_stderr=True, timeout=timeout)
 
     @classmethod
-    def delete_parameter(cls, options=None):
+    def delete_parameter(cls, options=None, timeout=None):
         """
         Deletes parameter from record.
         """
 
         cls.command_sub = 'delete-parameter'
 
-        result = cls.execute(cls._construct_command(options))
+        result = cls.execute(cls._construct_command(options), ignore_stderr=False, timeout=timeout)
```

Comment:
is this change intended here?

Reply:
it is, yes. similar commands had been updated with these arguments, and the lack of them ended up with failing mock tests.